packages/core/src/components/dropdown-button/dropdown-button.tsx

```diff
@@ -0,0 +1,145 @@
+import { Component, h, Prop, Watch, Element } from "@stencil/core";
+import clsx from "clsx";
+import { tabbable } from "tabbable";
+import { v4 as uuidv4 } from "uuid";
+
+function tabInPopup(host: Element, down: boolean) {
+  const tabs = tabbable(host);
+  const currentIndex = tabs.findIndex((e) => e === document.activeElement);
+  const nextIndex = down
+    ? currentIndex >= tabs.length - 1
+      ? 0
+      : currentIndex + 1
+    : currentIndex <= 0
+    ? tabs.length - 1
+    : currentIndex - 1;
+  (tabs[nextIndex] as HTMLElement)?.focus();
```

Comment:
Waarom cast jij naar `HTMLElement`? Het lijkt niet nodig:

![image](https://user-images.githubusercontent.com/9696261/126145837-69f9c53f-6ade-4012-aad8-793b70993232.png)

Reply:
En zie opmerking over foutafhandeling

Reply:
Is blijven staan na wat refactors, is inderdaad niet meer nodig hier.

Reply:
En is weg nu.